src/Todoist/Client.ts

```diff
@@ -33,10 +33,10 @@ export class Client {
     const opts: GoogleAppsScript.URL_Fetch.URLFetchRequestOptions = {
       method: 'post',
       contentType: 'application/x-www-form-urlencoded',
+      headers:{Authorization:"Bearer "+this.todoistToken},
```

Comment:
Use template literal, please.

```suggestion
      headers: { Authorization: `Bearer ${this.todoistToken}` }, // eslint-disable-line @typescript-eslint/naming-convention
```